DirectBoot/gradle.properties

```diff
@@ -1,20 +1,2 @@
-
```

Comment:
Should we keep these comments?

Reply:
Thanks. It dissppeared because of the AndroidX migration tool.  Fixed.